src/nxt_atomic.h

```diff
@@ -86,109 +86,7 @@ typedef volatile nxt_atomic_uint_t  nxt_atomic_t;
  */
 
 
-#elif (NXT_HAVE_XLC_ATOMIC) /* XL C/C++ V8.0 for AIX */
```

Comment:
I would merge this with the same commit that stops checking for these atomics, and with the same commit that removes support for this compiler.

I would split the commit that removes support for rare compilers, so that we remove support to one compiler at a time.